src/04-calendar/events.apib

```diff
@@ -109,6 +109,9 @@ Get details for a single calendar event.
             + starts_at: `2016-02-04T16:00:00+00:00` (string)
             + ends_at: `2016-02-04T18:00:00+00:00` (string)
             + location (string)
+            + related_work_order (object) - Only included with request parameter `includes=work_order`
```

Comment:
Q: do we want to link only 1? Or should we return a list of linked workOrders?
(for now, it is only possible to link 1 workOrder from the UI. But I assume we want to enable multiple draft WOs for a meeting in the future.)

Reply:
B: I'd rather say this fits in the existing `links` structure which holds a collection of interlinked entities.

A rather specific property for just (one, indeed) work order is pretty limiting and will get messy if we copy this pattern for more entity types.